lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -425,27 +419,29 @@ public TokenDetails authorize(TokenParams params, AuthOptions options) throws Ab
 		options = (options == null) ? this.authOptions : options.copy();
 		params = (params == null) ? this.tokenParams : params.copy();
 
+		tokenAuth.clear();
 		TokenDetails tokenDetails = tokenAuth.authorize(params, options);
```

Comment:
Shouldn't the private `TokenAuth#authorize` method instead support a new force param so that a new token can be generated without having to delete the old one first?